src/PackageVersions/Installer.php

```diff
@@ -48,15 +49,13 @@ final class Installer implements ComposerV2Plugin, EventSubscriberInterface
  */
 %s
 {
-    public const ROOT_PACKAGE_NAME = '%s';
     /**
-     * Array of all available composer packages.
-     * Dont read this array from your calling code, but use the \PackageVersions\Versions::getVersion() method instead.
-     *
-     * @var array<string, string>
-     * @internal
+     * @deprecated please use {@see \Composer\InstalledVersions::getRootPackage()} instead. The
+     *             equivalent expression for this constant's contents is
+     *             `\Composer\InstalledVersions::getRootPackage()['name']`.
+     *             This constant will be removed in version 2.0.0.
      */
-    public const VERSIONS          = %s;
```

Comment:
@ocramius It seems like removing this seems to break compatibility to [ocramius/proxy-manager:2.8](https://github.com/Ocramius/ProxyManager/blob/2.8.0/src/ProxyManager/Generator/Util/IdentifierSuffixer.php#L50)

I get the following error:

> !!  Fatal error: Uncaught Error: Undefined class constant 'VERSIONS' in /tmp/sulu-develop.localhost/vendor/ocramius/proxy-manager/src/ProxyManager/Generator/Util/IdentifierSuffixer.php:50

Reply:
Fairly sure that's my own fault inside `ocramius/proxy-manager` then: can you report an issue there, please?

Reply:
Note: this was `@internal` before, so I shouldn't have relied on it in first place.

Reply:
Oh I see make sense then that it need to be changed at the proxy-manager side.